techniques/general/G195.html

```diff
@@ -51,111 +49,110 @@
-         <h3>Example 3: Menus</h3>
+         <h3>事例 3: メニュー</h3>
          
-            <p>A web page includes an interactive menu with sub-menus. A user can move focus in the menu using the arrow keys. As focus moves, the currently focused menu item changes its background to a different color, which has a 3:1 contrast ratio with the surrounding items and a 4.5:1 contrast ratio with its own text.</p>
+            <p>サブメニューを持ったインタラクティブなメニューを含んだウェブページがある。利用者は矢印キーを使ってメニューの中でフォーカスを動かすことができる。フォーカスが動くと、フォーカスが当たっているメニュー項目はその背景を異なる色に変える。その色は周りの項目に対して 3:1 のコントラスト比を持ち、そのメニューのテキストの色と 4.5:1 のコントラスト比を持っている。</p>
          
       </section>
    </section>
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G149">G149: Using user interface components that are highlighted by the user agent when they receive focus</a></li>
       <li><a href="../general/G165">G165: Using the default focus indicator for the platform so that high visibility default focus indicators will carry over</a></li>
       <li><a href="../css/C15">C15: Using CSS to change the presentation of a user interface component when it receives focus</a></li>
       <li><a href="../client-side-script/SCR31">SCR31: Using script to change the background color or border of the element with focus</a></li>
       <li><a href="../css/C40">C40: Creating a two-color focus indicator to ensure sufficient contrast with all components</a></li>
       <li><a href="../css/C45">C45: CSS の :focus-visible を用いてキーボードフォーカス表示を提供する</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
-         <p>For each user interface component on the page that should receive keyboard focus:</p>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
+         <p>ページ上でキーボードフォーカスを受け取るべき各ユーザインタフェース コンポーネントについて:</p>
          <ol>
-            <li>Navigate to the component and check that it has a visible focus indicator.</li>
-            <li>Check that the focus indicator area is at least the size of a 1 CSS px border around the component.</li>
-            <li>If the focus indicator area is not at least equal to the area of a 1 CSS pixel border, check that it has an area of at least 4 CSS pixels along the shortest side of the component.</li>
-            <li>Check that the change of contrast of the indicator between focused and unfocused states has a ratio of 3:1 or more for the minimum focus indicator area.</li>
-            <li>If the focus indicator does not have 3:1 contrast ratio with its adjacent colors, check that it is at least 2px thick.</li>
+            <li>コンポーネントにナビゲートし、フォーカスインジケータが表示されていることをチェックする。</li>
+            <li>フォーカスインジケータの領域が、コンポーネントの周囲で少なくとも 1 CSS ピクセルのボーダーの大きさであることをチェックする。</li>
+            <li>フォーカスインジケータの領域が 1 CSS ピクセル以上のボーダーである場合、コンポーネントの短辺に沿って 4 CSS ピクセル以上の領域があることをチェックする。</li>
```

Comment:
原文では「If the focus indicator area is **not** at least equal to the area of a 1 CSS pixel border,」となっています。

```suggestion
            <li>フォーカスインジケータの領域が 1 CSS ピクセル以上のボーダーでない場合、コンポーネントの短辺に沿って 4 CSS ピクセル以上の領域があることをチェックする。</li>
```